api/bazel/repository_locations.bzl

```diff
@@ -40,8 +40,8 @@ REPOSITORY_LOCATIONS = dict(
     ),
     com_github_cncf_udpa = dict(
         sha256 = UDPA_SHA256,
-        strip_prefix = "udpa-" + UDPA_GIT_SHA,
-        urls = ["https://github.com/cncf/udpa/archive/" + UDPA_GIT_SHA + ".tar.gz"],
+        strip_prefix = "udpa-" + UDPA_RELEASE,
```

Comment:
Future work: switch to using string interpolation for these like `repository_locations.bzl`.